### PR TITLE
feat(components): Added JSON query components

### PR DESCRIPTION
**Pull request by Ark-kun**

### CODE REVIEW

File: components/json/Get_element_by_index/component.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  annotations:
+    author: Alexey Volkov <alexey.volkov@ark-kun.com>
+implementation:
+  container:
+    image: stedolan/jq:latest
+    command:
+    - sh
+    - -exc
+    - |
+      intput_path=$0
```
Comment (shaikmanu797) [inline components/json/Get_element_by_index/component.yaml:17]: Is the misspelling in input_path as intput_path intended in all of the files on the PR?

Comment (Ark-kun) [reply components/json/Get_element_by_index/component.yaml]: Fixed. Thanks.

**Pull request merged**